src/util/string.h

```diff
@@ -24,3 +25,18 @@ class StringCollator {
   private:
     QCollator m_collator;
 };
+
+class LinkFormatter {
+  public:
+    QString coloredLinkString(
+            const QColor color,
+            const QString* text,
+            const QString* baseUrl,
+            const QString* extUrl = nullptr) const {
+        return QString("<a style=\"color:%1;\" href=\"%2%3\">%4</a>")
```

Comment:
```suggestion
        return QStringLiteral("<a style=\"color:") + color.name() + QStringLiteral(";\" href=\"") + baseUrl +  extUrl +  QStringLiteral("\">") + text + QStringLiteral("</a>");
```
 The maximum boiler plate version but also the fastest.

Reply:
okay.
so `.arg(` makes the previous way slower?

Reply:
Yes, because the string is parsed for %1, than it is split and concatenated with the replacements. But this does not matter here at all.

Reply:
okay, I implemented the faster method. (fixed the clazy warnings as well)